htdocs/js/series-list.js

```diff
@@ -253,6 +253,7 @@ $(function () {
             this.success = $('#tests-success');
             this.warning = $('#tests-warning');
             this.failure = $('#tests-failure');
+            this.info = $('#tests-info');
```

Comment:
Hum, aren't you missing the HTML part of that? this element is needed in the menu markup.